File: Solutions/All/1980-FindUniqueBinaryString/FindUniqueBinaryString.cs
public class Solution
{
    public string FindDifferentBinaryString(string[] nums)
    {
        string newNum = "";

        for (int i = 0; i < nums.Length; i++)
        {
            if (nums[i][i] == '1')
            {
                newNum += '0';
            }
            else
            {
                newNum += '1';
            }
        }

        return newNum;
    }
}

File: Solutions/All/1980-FindUniqueBinaryString/FindUniqueBinaryString.ts
function findDifferentBinaryString(nums: string[]): string {
  let result = "";

  for (let i = 0; i < nums.length; i++) {
    if (nums[i][i] === "1") result += "0";
    else result += "1";
  }

  return result;
}
